client/dashboard/app/style.scss

```diff
@@ -41,3 +41,8 @@ body {
 a {
 	color: var( --wp-admin-theme-color );
 }
+
+// Hide the password reveal button in Edge.
+::-ms-reveal {
+	display: none;
+}
```

Comment:
Personally I would only do this on our specific password field. The user might be using Edge because they like this password behaviour. I think we have reason to disable it for our specific case because we implement our own, but otherwise I don't think it is right to override browser behaviour "just 'cause"

Reply:
Fair point 🙂